core/src/main/java/org/apache/calcite/rel/rules/MultiJoinProjectTransposeRule.java

```diff
@@ -128,7 +129,7 @@ protected LogicalProject getRightChild(RelOptRuleCall call) {
   // override JoinProjectTransposeRule
```

Comment:
Minor detail: maybe this is a good opportunity to replace all four comments `// override JoinProjectTransposeRule` in this class with the proper annotation `@Override`

Reply:
I'd happy to post a patch to fix it: I think @siddharthteotia has been really patient, so let's not abuse his time

Reply:
Sure, no problem.